README.md

```diff
@@ -1 +1,56 @@
-# Getting started with Cisco IOS-XE Smart licensing APIs
\ No newline at end of file
+# Getting Started with Cisco IOS-XE Smart licensing APIs
+
+Sample code in Python that shows how to call IOS-XE SLP REST APIs. You can automate the usage reporting workflows using these APIs.
+
```

Comment:
This is a sample python script demonstrating how to use the IOS-XE REST APIs for Smart Licensing workflows.
The workflows are part of the SLP (Smart Licensing using Policy) feature, which was introduced in IOS-XE release 17.3.2 and onwards. Using these APIs, the license usage reporting workflows can be automated.

Reply:
Will add this Sujay.  I'm going to condense this a little because the content is becoming repetitive. 
Sample Python script written in Python 3.8.5. The script demonstrates the use of IOS-XE SLP REST APIs to automate SLP workflows.
SLP was introduced in Cisco IOS-XE Amsterdam 17.3.2. 

For more information, see [About Smart Licensing Using Policy](ios-xe-smart-licensing-api/intro.md#about-smart-licensing-using-policy).